fecgov/fec-cms
# Fix/commitee type filter counts

## Summary 
- Resolves #2464

Fixes incorrect (double or quadrupled) filter counts in filter header in committee type filters for 
disbursements, audit-search and committees datatables.

Related PR: #2150 - Issues with committee filter counts came up in reviewing this PR.
## Impacted areas of the application

List general components of the application that this PR will affect:
       modified:  data/templates/partials/filters/spender-committee-types.jinja
       modified:   data/templates/partials/filters/audit-committee-types.jinja
       modified:   data/templates/partials/filters/committee-types.jinja

## How to test:
- Run this branch locally : `f`ix/committee-type-filter-counts`
- Test Committee Type filters to make sure that the counts shown to the right of the header are correct: Also spot check data result counts (not filter counts)  against production to confirm that this change does not effect returned data in any way.
- Check filter tags in datatable panel to make sure these behave as expected
- Possible pages to test:
- http://127.0.0.1:8000/legal-resources/enforcement/audit-reports/
(Tests `audit-committee-types.jinja`)
- https://www.fec.gov/data/committees/
(Tests: `committee-types.jinja`)
- http://127.0.0.1:8000/data/disbursements/
(Tests `spender-committee-types.jinja`)
